### PR TITLE
Rebuild libsingular_julia

### PR DESCRIPTION
**Pull request by fingolfin**
... for compatibility with latest libjulia 1.8

### CODE REVIEW

**Comment by giordano** [comment]
It's a bit unfortunate we have to rebuild all tarballs when we only really need the ones for v1.7. This isn't going to be sustainable in the long run. We should devise a way to update only some tarballs

**Comment by fingolfin** [comment]
Agreed. Basically if one could specify something like "skip build" but with a filter regex so that building is only skipped for matching platforms... and then for the skipped platforms we use the binaries from the previous build with the same version

**Comment by fingolfin** [comment]
Of course another option in the special case of rebuilding for Julia 1.8 would be this: increment the version; set the julia compat to >= 1.8; and then build a JLL with only 1.8 support.

of course if we then later need an actual new version also covering 1.6, one has to be very careful to restore everything. That's bad because it is super easy to get wrong, so it's not really a good long term solution.

I guess reproducible builds would also help: if the artifact tarballs had identical sha sum, then that's also fine...

**Comment by benlorenz** [comment]
For libpolymake_julia I have new 0.6.1 version almost ready (up to some final tests) and I will create a separate PR for that patch-level bump with some minor improvements either today or tomorrow (i.e. we can skip the rebuild for this one).

**Comment by fingolfin** [comment]
We actually may not neeed this anyway... converting to draft for now until I fix GAP.jl to allow our tests to proceed further

**Comment by benlorenz** [comment]
At least for polymake the updated libcxxwrap alone did not fix the test failures for julia nightly, the failure moved from precompiling CxxWrap to precompiling Polymake and the types in the error message changed. 
A local rebuild of both `libpolymake_julia` and `libcxxwrap_julia` did fix this error.

**Comment by fingolfin** [comment]
Ok then let's do this anyway

**Comment by benlorenz** [comment]
The rebuild for libpolymake_julia is not needed anymore (was done with #4124), so you can readd that deleted line.

File: L/libpolymake_julia/build_tarballs.jl [hunk 1]
```diff
@@ -3,7 +3,6 @@
 using BinaryBuilder, Pkg
 using Base.BinaryPlatforms
```
Comment (fingolfin) [inline L/libpolymake_julia/build_tarballs.jl:5]:
```suggestion


```

**Pull request merged**